src/Chord.hs

```diff
@@ -35,7 +35,7 @@ data Root
   = Root Note Accidental
 
 instance Show Root where
-  show (Root note acc) = (show note) ++ (show acc)
+  show (Root note acc) = show note ++ show acc
```

Comment:
A good rule of thumb to keep in mind is "function application binds tighter than any infix operator."